blog/content/post/2024/dbatools-repl-setup/index.md

```diff
@@ -74,34 +124,70 @@ Articles      : {}
 Subscriptions : {}
 ```
 
-## Add article
+More information on this command is available in the dbatools help documentation here:
+
+- [New-DbaReplPublication](https://dbatools.io/New-DbaReplPublication)
+
+## Add an Article
+
+Once we have a publication in place it's time to add articles, these are the objects we want to replicate from one database to another. I'm going to use `Add-DbaReplArticle` to add the `SalesLT.Customer` tables to my `testpub` publication.
+
+You'll also notice in this case I've added a `Filter` this is a `WHERE` clause that will filter the rows that are replicated. I think this is a really cool feature, for this setup only rows where `lastname = 'gates'` will be replicated to the subscriber.
 
 ```PowerShell
 $article = @{
     SqlInstance = 'sql1'
     Database    = 'AdventureWorksLT2022'
     Publication = 'testpub'
-    Schema      = 'salesLT'
-    Name        = 'customer'
+    Schema      = 'SalesLT'
+    Name        = 'Customer'
     Filter      = "lastname = 'gates'"
 }
 Add-DbaReplArticle @article
 ```
 
+Executing that returns the following output showing that my article has been added successfully.
+
 ```text
 ComputerName      : sql1
 InstanceName      : MSSQLSERVER
 SqlInstance       : [sql1]
 DatabaseName      : AdventureWorksLT2022
 PublicationName   : testPub
-Name              : customer
+Name              : Customer
 Type              : LogBased
 VerticalPartition : False
 SourceObjectOwner : SalesLT
 SourceObjectName  : Customer
 ```
 
-## Add a subscrition
+If I review the properties of the publication from SSMS I can see that the article has been added successfully.
+
+{{<
+    figure src="article.png"
+    alt="Properties pane for the testPub publication showing articles"
+>}}
+
+If I change to look at the `Filter Rows` page I can see the filter I specified is in place.
+
+{{<
+   figure src="articlefilter.png"
+   alt="Filter Rows page of the testPub publication properties window"
+>}}
+
+More information on this command is available in the dbatools help documentation here:
+
+- [Add-DbaReplArticle](https://dbatools.io/Add-DbaReplArticle)
+
+Alright, the final piece of this puzzle is to add a subscription.
```

Comment:
Double-check if would make sense to move it to after the next title. Within the `Add a Subscription`